js/main.js

```diff
@@ -116,25 +154,125 @@ const createPin = (array) => {
+      }
+      if (!featuresArr.includes(`conditioner`)) {
+        featuresNode.querySelector(`.popup__feature--conditioner`).style = `display: none`;
+      }
+    }
+  }
+
+  return featuresNode;
+};
+
+const createCard = (object) => {
+  const cardElement = mapCardTemplate.cloneNode(true);
+  const photoSource = cardElement.querySelector(`.popup__photos`);
+  const photoImg = photoSource.querySelector(`.popup__photo`);
+  const featuresSource = cardElement.querySelector(`.popup__features`);
+  delChild(cardElement, featuresSource);
+  delChild(photoSource, photoImg);
+  cardElement.querySelector(`.popup__title`).textContent = object.offer.title;
+  cardElement.querySelector(`.popup__text--address`).textContent = object.offer.address;
+  cardElement.querySelector(`.popup__text--price`).textContent = `${object.offer.price} ₽/ночь`;
+  cardElement.querySelector(`.popup__type`).textContent = translateHouseType(object.offer.type);
+  cardElement.querySelector(`.popup__text--capacity`).textContent = `${object.offer.rooms} комнаты для ${object.offer.guests} гостей`;
+  cardElement.querySelector(`.popup__text--time`).textContent = `Заезд после ${object.offer.checkin}, выезд до ${object.offer.checkout}`;
+  cardElement.querySelector(`.popup__description`).textContent = object.offer.description;
+  cardElement.querySelector(`.popup__avatar`).src = object.author.avatar;
+  const featuresNode = cardElement.insertBefore(createFeaturesElements(object.offer.features), cardElement.querySelector(`.popup__description`));
+  const photosNode = cardElement.appendChild(createPhotosElements(object.offer.photos, photoSource));
+  disableNode(featuresNode, `li`);
+  disableNode(photosNode, `img`);
+  disableTextElement(cardElement, `p`);
+  disableTextElement(cardElement, `h3`);
+  disableTextElement(cardElement, `h4`);
+
+  return cardElement;
+};
+
+const createPinsNodeFragment = (pinsArr) => {
   const fragment = document.createDocumentFragment();
-  for (let i = 0; i < pin.length; i++) {
-    fragment.appendChild(createPin(pin[i]));
+  for (let i = 0; i < pinsArr.length; i++) {
+    fragment.appendChild(createPin(pinsArr[i]));
   }
 
   return fragment;
 };
 
+const createСardFragment = (cardObj) => {
+  const fragment = document.createDocumentFragment();
+  fragment.appendChild(createCard(cardObj));
+  return fragment;
+};
+
 const addNodeFragment = (element) => {
   mapPinsNode.appendChild(element);
 };
 
+const addCardFragment = (element) => {
+  mapNode.insertBefore(element, mapFiltersNode);
+};
+
 const initPinsScreen = () => {
-  const pinsDataArray = createDataArray(PIN_AMOUNT);
-  const pinsNodesFragment = createNodeFragment(pinsDataArray);
+  const pinsDataArray = createDataArray(PINS_AMOUNT);
+  const pinsNodesFragment = createPinsNodeFragment(pinsDataArray);
+  const cardNodesFragnment = createСardFragment(pinsDataArray[0]);
 
+  addCardFragment(cardNodesFragnment);
   addNodeFragment(pinsNodesFragment);
```

Comment:
Названия функций не говорят о том, куда они добавляют фрагменты, также эти функции выполняют очень простую задачу и вызываются всего единожды, понятнее было бы написать без них
```suggestion
  mapNode.insertBefore(cardNodesFragnment, mapFiltersNode);
  mapPinsNode.appendChild(pinsNodesFragment);
```